### PR TITLE
Add every favicon permutation under the sun

### LINKED ISSUE #333: Please update favicon
Please update the favicon on clojars.org to correspond with the logo here: https://clojars.org/images/clojars-logo-small.png.

### PR DESCRIPTION
**Pull request by danielcompton**
I used http://realfavicongenerator.net to generate every possible favicon that you could want. I used the 170x170 `clojars-logo-big.png`. A higher resolution image would be nicer, but I think it looks good like this. Happy to rerun it again if you've got a larger source image. The query parameter on images is to bust browser caches.

The [realfavicongenerator FAQ](http://realfavicongenerator.net/faq#.VZxCbniIfhs) explains all about the different files. Let me know if you've got any questions though.

Fixes #333

### CODE REVIEW

**Comment by danielcompton** [comment]
I left the original favicon.ico, but could update that too if you like.

File: src/clojars/web/common.clj [hunk 1]
```diff
@@ -92,15 +92,33 @@
...
+    [:link {:rel "apple-touch-icon" :sizes "152x152" :href "/apple-touch-icon-152x152.png?v=47K2kprJd7"}]
+    [:link {:rel "icon" :type "image/png" :href "/favicon-32x32.png?v=47K2kprJd7" :sizes "32x32"}]
+    [:link {:rel "icon" :type "image/png" :href "/favicon-96x96.png?v=47K2kprJd7" :sizes "96x96"}]
+    [:link {:rel "icon" :type "image/png" :href "/favicon-16x16.png?v=47K2kprJd7" :sizes "16x16"}]
+    [:link {:rel "manifest" :href "/manifest.json?v=47K2kprJd7"}]
+    [:link {:rel "shortcut icon" "/favicon.ico?v=47K2kprJd7"}]
+    [:meta {:name "apple-mobile-web-app-title" :content "Clojars"}]
+    [:meta {:name "application-name" :content "Clojars"}]
+    [:meta {:name "msapplication-TileColor" "#da532c"}]
+    [:meta {:name "msapplication-TileImage" "/mstile-144x144.png?v=47K2kprJd7"}]
```
Comment (tobias) [inline src/clojars/web/common.clj]: Lines 115 is missing an `:href` key, and 118 and 119 are missing a `:content` key. Would you mind fixing those?

**Comment by tobias** [comment]
I had no idea favicons had become so complicated, thanks for this. If you'll fix the meta issues, I'll pull this and see how the full icon looks for the common cases.

**Comment by danielcompton** [comment]
I've fixed this, sorry about that.

**Comment by tobias** [comment]
This looks good! I don't have a larger logo, but @devn may be able to provide one. I'm comfortable with the way they look now, but if you want to wait and see if we can get a higher-res image, we can. 

As for needing a favicon.ico, I defer to your judgment.

**Comment by devn** [comment]
Unfortunately I don't have access to a higher resolution image. Sorry :\

**Comment by tobias** [comment]
@devn - no worries, thanks for letting us know.

**Comment by danielcompton** [comment]
I think it would be better to keep the current favicon.ico, it will look better than scaling a large image to a very small one.

**Comment by tobias** [comment]
Sounds good to me - I'll merge this, but may wait to see if we can get a few other fixes in before I release.

**Pull request merged**